controllers/book.controller.js

```diff
@@ -37,6 +39,7 @@ module.exports.addBook = (req, res) => {
 module.exports.deleteBook = (req, res) => {
     Book.deleteOne({ _id: req.params.id })
         .then((mres) => {
+            Logger.addLog('books',`${req.userData.firstname} ${req.userData.lastname} deleted ${req.body.title}`);
```

Comment:
You've used 'books' here when on other cases you've used 'book'. Let's go with just either 'book' or 'books' and keep it consistent.

Reply:
done!